src/components/implementation/tests/unit_constant_value/unit_constant_value.c

```diff
@@ -3,14 +3,12 @@
 int
 main(void)
 {
-	if ( NUM_CPU == 4 )
-	{
+	if (NUM_CPU == 4){
 			printc("SUCCESS: CONSTANT VALUE(NUM_CPU) has been set to 4 \n");
 	}
-	else
-	{
+	else{
```

Comment:
See my comment in the previous review. There's special formatting for if, else.

Reply:
add the space